### PR TITLE
Fix code quality issues

### PR DESCRIPTION
**Pull request by wizard-28**
# Purpose

Lot of code quality issues in [Codacy](https://app.codacy.com/gh/pacstall/pacstall/issues?bid=25100148&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6WyJQeXRob24iXX0seyJpZCI6IkNhdGVnb3J5IiwidmFsdWVzIjpbXX0seyJpZCI6IkxldmVsIiwidmFsdWVzIjpbXX0seyJpZCI6IlBhdHRlcm4iLCJ2YWx1ZXMiOltdfSx7ImlkIjoiQXV0aG9yIiwidmFsdWVzIjpbXX1d)

# Approach

Tries to fix them all.

# Progress

Make a checklist of your progress

- [x] Example checklist

# Addendum

Left some issues, as they are false positives.

### CODE REVIEW

**Review by Henryws — APPROVED** [review]

**Review by NarutoXY — APPROVED** [review]

File: install.py [hunk 1]
```diff
@@ -131,13 +131,10 @@ def download(url: str, filepath: str = os.getcwd()) -> None:
...
-    exit(1)
+    sys.exit(1)
 
-if not Popen(
-    "find -H /var/lib/apt/lists -maxdepth 0 -mtime -7",
-    shell=True,
-    stdout=PIPE,
-).stdout:
+
+if not check_output(["find", "-H", "/var/lib/apt/lists", "-maxdepth", "0", "-mtime", "-7"]):
```
Comment (D-Brox) [inline install.py:137]: Is this faster than using `glob` and `time`?

**Review by D-Brox — COMMENTED** [review]
Why did we port `install.py`? Shouldn't pip deal with the installation?

File: install.py (hunk 1 above)
Comment (wizard-28) [reply install.py:137]: I couldn't come up with a python alternative for this line... So I used subprocess. Can you suggest one?

Comment (D-Brox) [reply install.py:137]: Or using `os.stat`

Comment (D-Brox) [reply install.py:137]: Or using `os.stat()`

Comment (wizard-28) [reply install.py:137]: Can you write the code snippet.

Comment (D-Brox) [reply install.py:137]:
```python
if [f for f in glob.glob('/var/lib/apt/lists/*') if os.stat(f).st_mtime >  604800]:
     print(1)
```

Comment (wizard-28) [reply install.py:137]: Ouput: `1` means it's older than 1 week right?

Comment (D-Brox) [reply install.py:137]: Yes, all files are older than 1 week in this case

Comment (wizard-28) [reply install.py:137]: I just updated but it still outputs `1`...

Comment (D-Brox) [reply install.py:137]:
```python
import glob
import time

if not [f for f in glob.glob('/var/lib/apt/lists/*') if os.stat(f).st_mtime < time.time()  604800]:
     print(1)
```

Comment (wizard-28) [reply install.py:137]: Your method ran faster
```
Summary
  'python3 g.py' ran
    1.15 ± 0.03 times faster than 'python3 f.py'
```

Implementing it

**Review by Henryws — APPROVED** [review]

**Review by D-Brox — APPROVED** [review]
While I still think we should make a python module for pacstall instead of scripts, I'll approve for now

**Pull request merged**